http-core/src/main/scala/org/apache/pekko/http/javadsl/model/ContentType.scala

```diff
@@ -61,4 +61,9 @@ trait ContentType {
    * Returns the charset if this ContentType is non-binary.
    */
   def getCharsetOption: Optional[HttpCharset]
+
+  /**
+   * Returns the string representation of this ContentType
+   */
```

Comment:
can you add `@since 1.1.1` to the 2 new public functions?

Reply:
if we do go the 1.2.0 route, we will need to delay this PR merge until we get to create the 1.1.x branch and the CI jobs related to it

Reply:
Thats fine, its not urgent

Reply:
* ok - let's go the 1.2.0 route
* I'll try to set up the CI builds over the next few days

Reply:
Maybe just override the `toString` method?